producer_test.go

```diff
@@ -1248,6 +1248,57 @@ func testTopicProducerMgr(t *testing.T, dynamic bool) {
 	wg.Wait()
 }
 
+func TestRemoveUnusedProducerAsync(t *testing.T) {
```

Comment:
Need more test cases to test the failed while query lookup

Reply:
@Patty-Chen

Reply:
add test for queryLookupd